lib/util/getAutorestConfigInternal.ts

```diff
@@ -46,7 +46,7 @@ const getAutorest = async (version: string | undefined, rfs?: IFileSystem, readm
 };
 
 const getAutorestCoreVersion = async (_args?: any) => {
-  return "latest-installed";
+  return "3.0.6375";
```

Comment:
Is it conflict with the installed version is `3.0.6335`?

Reply:
This is aligned with the version in alps.